values.yaml

```diff
@@ -28,6 +28,21 @@ mastodon:
       enabled: true
     assetsPrecompile:
       enabled: true
+    # Upload website assets to S3 before deploying.
+    # Whenever there is an update to Mastodon, sometimes there are assets files
+    # that are renamed. As the pods are getting redeployed, and old/new pods are
+    # present simultaneously, that old asset files are requested from pods that
+    # don't have them anymore. Uploading asset files to S3 in this manner solves
+    # this potential conflict.
```

Comment:
@renchap Please check my description here to make sure it's accurate, or if needs to be reworded.